cmake/projects/OpenSSL/schemes/url_sha1_openssl.cmake.in

```diff
@@ -33,6 +33,11 @@ else()
   set(configure_command "./config")
 endif()
 
+# Pass fPIC through
+if(";${CMAKE_CXX_FLAGS};" MATCHES ";-fPIC;")
```

Comment:
I see that you have pushed new commit. If you're done, let me know about it because I have no idea that you've finished updates or it's just pushed a work-in-progress stuff:
* https://github.com/ruslo/hunter/blob/master/.github/CONTRIBUTING.rst
* https://github.com/ruslo/hunter/wiki/dev.contribution